container.go

```diff
@@ -0,0 +1,421 @@
+package llblib
+
+import (
+	"context"
+	goerrors "errors"
+	"io"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/moby/buildkit/client/llb"
+	"github.com/moby/buildkit/frontend/gateway/client"
+	gateway "github.com/moby/buildkit/frontend/gateway/client"
+	"github.com/moby/buildkit/solver/pb"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh/terminal"
+	"golang.org/x/sync/errgroup"
+	"golang.org/x/sys/unix"
+)
+
+const unsetArgsSentinel = "/dev/null"
```

Comment:
Would an empty string work here?

Reply:
I think so, the ExecOp.Validate function only cares that the llb.Args is not an empty list, so a list with one empty string seems to work.  Not sure if that will be more/less confusing than something like `/dev/null` but I went with it for now.